### PR TITLE
Antrea 1.5.3 package bump

### PR DESCRIPTION
**Pull request by knabben**
### Which issue(s) this PR fixes

Fixes #

### Release note

```release-note
Bumping Antrea component to version 1.5.3
```

### PR Checklist

- [ ] Squash the commits into one or a small number of logical commits
      
- [ ] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [ ] Ensure PR contains terms all contributors can understand and links all contributors can access

### CODE REVIEW

File: addons/controllers/antrea/antreaconfig_util.go [hunk 1]
```diff
@@ -37,6 +37,13 @@ type antreaProxy struct {
 	ProxyLoadBalancerIPs bool     `yaml:"proxyLoadBalancerIPs,omitempty"`
 }
 
+type antreaFlowExporter struct {
+	Address       string `yaml:"address,omitempty"`
```
Comment (edwardbadboy) [inline addons/controllers/antrea/antreaconfig_util.go:41]: I have a suggestion for the names:
- Address -> CollectorAddress
- ExportTimeout -> ActiveFlowTimeout
- IdleTimeout  -> IdleFlowTimeout

**Review by edwardbadboy — COMMENTED** [review]
Suggested a small change in variable names.

**Comment by knabben** [comment]
//usebom:bom-generation/415313579022001923/tkg-compatibility

**Review by edwardbadboy — APPROVED** [review]

**Comment by knabben** [comment]
//usebom:bom-generation/416531951856447326/tkg-compatibility

**Review by vuil — APPROVED** [review]
lgtm. Please add a release note about the version bump. 
Holding for downstream verification.

**Pull request merged**